cli/README.md

```diff
@@ -55,6 +55,28 @@ Noora().yesOrNoChoicePrompt(
 
 Noora CLI provides a comprehensive set of components for building beautiful terminal interfaces. Check out [our documentation](https://noora.tuist.dev) to see all available components and their usage.
 
+### Live-updating tables
```

Comment:
We have a documentation site under `docs/` the documentation of components should go there following the same structure and content style as the other components.